test/stages/stage1-namespace.tf

```diff
@@ -3,7 +3,8 @@ module "gitops_namespace" {
 
   gitops_config = module.gitops.gitops_config
   git_credentials = module.gitops.git_credentials
-  name = var.namespace
+  #name = var.namespace
```

Comment:
is this still needed?  If not, let's remove it.

Reply:
Removed